packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -103,21 +106,18 @@ const GetNavItemContentStyles = ({
       backgroundColor: !isDisabled
         ? odysseyDesignTokens.HueNeutral50
         : "inherit",
+      color: isDisabled
```

Comment:
This color declaration may be unnecessary. Did you find that hovering a disabled item was changing the color? I did not see that behavior

Reply:
yes, I did. I can double check again and attach a screenshot

Reply:
I just verified it and I cannot replicate it anymore. removing this check.